topo/node/srl/configpush_fail_cli.cfg

```diff
@@ -0,0 +1,2 @@
+set / system inftion location "wrong command"
```

Comment:
this is such a small nit but I couldn't help myself from pointing out that the files are named configpush_{} and configpush_{}_cli.cfg and one is fail and the other is failure while success is consistent

Reply:
good call, I missed that 
fixed in 63afc4c30438bfd7f72ced7952f4956b9d4e57ac